repos/system_upgrade/common/actors/checkipaserver/libraries/checkipaserver.py

```diff
@@ -12,15 +12,14 @@ def ipa_inhibit_upgrade(ipainfo):
     """
     entries = [
         reporting.Title(
-            "ipa-server does not support in-place upgrade from RHEL 7 to 8"
+            "ipa-server does not support in-place upgrade"
         ),
         reporting.Summary(
             "An IdM server installation was detected on the system. IdM "
-            "does not support in-place upgrade to RHEL 8. Please follow "
-            "the migration guide lines."
+            "does not support in-place upgrade."
         ),
         reporting.Remediation(
-            hint="Please follow the IdM RHEL 7 to 8 migration guide lines."
+            hint="Follow the IdM RHEL migration guide lines."
         ),
         reporting.ExternalLink(
             url=MIGRATION_GUIDE, title="Migrating IdM from RHEL 7 to 8",
```

Comment:
Unfortunately we don't have any 8->9 migration guide yet.

Reply:
After a discussion with @Rezney, we've arrived at conclusion that we will track the missing documentation string internally and modify the code accordingly once the guide becomes available. As we would like to avoid providing a link to incorrect migration guide when upgrading from 8 to 9, could you please change the code according to the suggestions above and use `TBA` for the missing upgrade link?